utils/bcm27xx-eeprom/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bcm27xx-eeprom
+PKG_VERSION:=a9ca308223c1d0426b9ab320696b95954078c3b4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/raspberrypi/rpi-eeprom/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=072dbbd4b53c2c0b5764ff628f63cc8d679a99cfe84d8f50acea06922084515e
+
+PKG_MAINTAINER:=Álvaro Fernández Rojas <noltari@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+TAR_OPTIONS:=--strip-components 1 $(TAR_OPTIONS)
+TAR_CMD=$(HOST_TAR) -C $(1) $(TAR_OPTIONS)
+
+define Package/bcm27xx-eeprom
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=bcm27xx-userland +blkid +pciutils +python
```

Comment:
does this really require Python2? Would something like `python3-light` work as well?

Reply:
I tried, and light isn't enough for both python 2 and 3.
There were some other modules required.

Reply:
There are plenty of small Python modules ported to OpenWrt and the [import](https://github.com/raspberrypi/rpi-eeprom/blob/master/rpi-eeprom-config#L7) don't look to complicated. Please check if it's possible to reduce the size.

I'm against merging a Python2 package, can you please request upstream to change it to Python3 or add another patch?

Reply:
I just tried again with python3-light and it seems to work fine, so I've added a patch that switches to Python 3 in my latest update.